setup.py

```diff
@@ -53,6 +53,6 @@
     setup_requires=["cffi"],
     ext_package="metaswitch.common",
     cffi_modules=["cffi_build.py:ffi"],
-    install_requires=["py-bcrypt", "pycrypto==2.6.1", "pyzmq==15.2", "cffi==1.5.2"],
+    install_requires=["py-bcrypt", "pycrypto==2.6.1", "pyzmq==15.2", "cffi==1.5.2", "monotonic==0.6"],
```

Comment:
It doesn't look like it: https://docs.python.org/2/distutils/setupscript.html#relationships-between-distributions-and-packages. [PEP 440](http://legacy.python.org/dev/peps/pep-0440/) looks like a step in the right direction but it's not clear that anything actually happened as a result of it being accepted.